src/status_im/browser/core.cljs

```diff
@@ -179,7 +179,7 @@
                                assoc
                                :url (str gateway path)
                                :resolving? false)
-                       (assoc-in [:browser/options :resolved-ens host] gateway))})))
+                       (assoc-in [:browser/options :resolved-ens host] (subs gateway 8)))})))
```

Comment:
Magic number? Can we use a separate function for that?

Reply:
it is not a real fix anyway